packages/installations/e2e/installations.modular.e2e.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const jwt = require('jsonwebtoken');
+
+const ID_LENGTH = 22;
+const PROJECT_ID = 448618578101; // this is "magic", it's the react-native-firebase-testing project ID
+
+describe('installations() modular', function () {
+  describe('getId()', function () {
```

Comment:
In the modular.e2e.js files, we keep both the modular and  non-modular in the same file. In future steps of the migration, we will probably remove the non-modular file, and have both files make sure that we don't forget to migrate a test.

Reply:
Added v8 compatibility tests :D